homework.md

```diff
@@ -4,83 +4,84 @@
 
 What is 2 + 2?
 
-5
+4.
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White.
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240.
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hypertext Markup Language.
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Describes the presentation of a webpage or document.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
-
+h1 {
 }
 ```
 
```

Comment:
On number 6 i think you don't need to include these comas ,,,

Reply:
Actually that is best practice because it indicates where the **css** starts and ends when he includes the back-ticks at the end of the  **css code**

Reply:
That is given as a guidance, it was there before I changed the answer, thanks Ntshembo, will take note of that,